.cargo/config-ci.toml

```diff
@@ -2,8 +2,15 @@
 debug = "none"
 incremental = false
 
+[target.x86_64-unknown-linux-gnu]
+rustflags = ["-Clink-arg=-fuse-ld=mold"]
+
+[target.aarch64-unknown-linux-gnu]
+rustflags = ["-Clink-arg=-fuse-ld=mold"]
+
+[target.x86_64-pc-windows-msvc]
+linker = "rust-lld.exe"
```

Comment:
Nit: can't we just use normal `lld-link` here or is the Rust distribution a big deviation from it?

Reply:
`rust-lld` is what Rust is [using by default for `x86_64-unknown-linux-gnu` on Nightly](https://blog.rust-lang.org/2024/05/17/enabling-rust-lld-on-linux.html) and [considering defaulting to on Windows](https://github.com/rust-lang/rust/issues/71520), so this is the better‐paved path; it also matches the majority of the recommendations I saw online. I tested with `lld-link.exe` and it seemed like it was slower rather than faster, but see: incredibly noisy measurements. The Windows GitHub Actions builders have LLVM 14/15 installed, which is considerably older than Rust’s LLVM, though I don’t know if it was using `lld-link.exe` from there or from Rust.